dbms/src/IO/ReadHelpers.cpp

```diff
@@ -569,6 +572,74 @@ void readDateTimeTextFallback(time_t & datetime, ReadBuffer & buf)
 }
 
 
+void skipJSONFieldPlain(ReadBuffer & buf, const String & name_of_filed)
```

Comment:
Need to pass StringRef, because copying to string is needed only for exceptions.